buildSrc/build.gradle

```diff
@@ -22,15 +22,15 @@ repositories {
 }
 
 dependencies {
-  compile (group: 'org.apache.geode', name: 'geode-junit', version: '1.3.0') {
+  compile ('org.apache.geode:geode-junit:1.3.0') {
```

Comment:
more of curiosity than actually caring, but how do we feel about the `()`?  Noticed below they are not used.

Reply:
Huh.  Parens are supposed to be added by `spotlessGroovyGradleApply`, but it looks like `buildSrc` is considered part of the `rootProject` by Gradle, rather than a `subproject` of the root.

I'll open a separate PR to fix that.  Thanks for noticing!